nvflare/app_common/workflows/cross_site_model_eval.py

```diff
@@ -432,22 +441,37 @@ def _accept_val_result(self, client_name: str, result: Shareable, fl_ctx: FLCont
 
             self._val_results[client_name][model_owner] = {}
```

Comment:
This line is not safe - the _val_results dict may not have item for client_name yet. Change to:

if not client_name in self._val_results:
    self._val_results[client_name] = {}

Reply:
Good catch! Changed.